frontend/test/__support__/commands.js

```diff
@@ -134,6 +133,42 @@ Cypress.Commands.add("updateCollectionGraph", (groupsCollectionObject = {}) => {
   );
 });
 
+Cypress.Commands.add(
+  "sandboxTable",
+  ({
+    attribute_remappings = {},
+    card_id = null,
+    group_id = 4,
+    table_id = 2,
```

Comment:
It seems like we should always pass a table ID (and probably group id too) rather than providing defaults.

Reply:
Or, if we do want a default group, we should pull it from `USER_GROUPS`.

Reply:
I wanted to pull it from `USER_GROUPS` but for that, I need to extract both `USERS` and `USER_GROUPS` into a new file - one that can be imported both into `commands.js` and `cypress.js`.

I thought it's best to leave it for a separate repro.

Reply:
Not urgent, but I also don't think there's much downside to cluttering the diff with moving those constants over if it relates to this change.

But yeah, I think `table_id` and maybe `attribute_remappings` don't really need defaults since they always need to specified anyways I think?